vault/resource_consul_secret_backend_role_test.go

```diff
@@ -96,7 +141,7 @@ func testAccConsulSecretBackendRoleCheckDestroy(s *terraform.State) error {
 	return nil
 }
 
-func testConsulSecretBackendRole_initialConfig(backend, name, token string, withPolicies, withRoles bool) string {
+func testConsulSecretBackendRole_initialConfig(backend, name, token string, withPolicies, isVersion111OrNewer bool) string {
```

Comment:
nit: Should we rename this variable to be more generic, maybe something like `isAboveVersionThreshold` or something?

Reply:
Yeah that sounds better, changing it now.